app/src/test/java/com/stevesoltys/seedvault/restore/install/ApkBackupRestoreTest.kt

```diff
@@ -49,7 +49,7 @@ internal class ApkBackupRestoreTest : TransportTest() {
 
     @Suppress("Deprecation")
     private val legacyStoragePlugin: LegacyStoragePlugin = mockk()
-    private val storagePlugin: StoragePlugin = mockk()
+    private val storagePlugin: StoragePlugin = mockk(relaxed = true)
```

Comment:
That *does* fix the test, but I think it would be better to add a return value explicitly where needed:

example:

`every { storagePlugin.providerPackageName } returns "org.example"`

This way, you can also add a new test that checks that if the returned package name would normally be restored, it won't get restored if returned by the storagePlugin.